bamboo/integration_tests/test_integration_lenet.py

```diff
@@ -14,6 +14,15 @@
 sys.path.insert(0, os.path.join(os.path.dirname(current_dir), 'common_python'))
 import tools
 
+# Applications directories
+bamboo_dir = os.path.dirname(current_dir)
+lbann_dir = os.path.dirname(bamboo_dir)
+sys.path.insert(0, os.path.join(lbann_dir, 'applications', 'vision'))
+import data.mnist
+
+# Python directory
+sys.path.insert(0, os.path.join(lbann_dir, 'python'))
```

Comment:
Importing the Python package in the source directory will not work since it doesn't have access to the LBANN executable or the Protobuf messages. You need to import the Python package from the install directory.

Reply:
Okay.  Fixed this.